packages/flutter/lib/src/widgets/autocomplete.dart

```diff
@@ -353,6 +366,9 @@ class _RawAutocompleteState<T extends Object> extends State<RawAutocomplete<T>>
   // Called when _textEditingController changes.
   Future<void> _onChangedField() async {
     final TextEditingValue value = _textEditingController.value;
+    if (widget.showOptionsViewOnUncompletedOptions){
+      _updateOptionsViewVisibility();
```

Comment:
How does this affect synchronous values of optionsBuilder? optionsViewBuilder could be called twice back-to-back. Is that something we should avoid or it shouldn't matter?

Reply:
If `optionsBuilder` is synchronous, and `showOptionsViewOnPendingOptions` is set to true, then the options view will just be rebuilt to display its current contents.

I've added a note for users to ignore `showOptionsViewOnPendingOptions` if their `optionsBuilder` is synchronous.